gnocchi/rest/aggregates/api.py

```diff
@@ -206,7 +183,14 @@ class AggregatesController(rest.RestController):
 
     @pecan.expose("json")
     def post(self, start=None, stop=None, granularity=None,
-             needed_overlap=None, fill=None, groupby=None, **kwargs):
+             needed_overlap=None, fill=None, groupby=None, use_history=None,
+             **kwargs):
+
+        if use_history is None:
+            use_history = pecan.request.conf.api.group_measures_using_history
```

Comment:
Yeah I don't think a global option should change a default behaviour.

Reply:
Ok, no problem, I will remove it.